src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -33,6 +32,29 @@ private void initFirstRun(Connection connection) throws SQLException, IOExceptio
 		this.context.setPosition(pos);
 	}
 
+	private void setupFilter(MaxwellFilter filter, MaxwellConfig config) {
+		if (!config.include_databases.isEmpty()) {
+			for (String result : config.include_databases.split(",")) {
+				filter.includeDatabase(result);
+			}
+		}
+		if (!config.include_tables.isEmpty()) {
+			for (String result : config.include_tables.split(",")) {
+				filter.includeTable(result);
+			}
+		}
+		if (!config.exclude_databases.isEmpty()) {
+			for (String result : config.exclude_databases.split(",")) {
+				filter.excludeDatabase(result);
+			}
+		}
+		if (!config.exclude_tables.isEmpty()) {
+			for (String result : config.exclude_tables.split(",")) {
+				filter.excludeTable(result);
+			}
+		}
+	}
```

Comment:
generally I've been putting code that glues config to maxwell into the `Context` object.  If you want to move it, great.  otherwise I'll move this code over before merging...